project-base/src/Shopsys/ShopBundle/Form/Front/Order/OrderFlow.php

```diff
@@ -164,7 +164,7 @@ private function changeRequestToStep(StepInterface $step)
         $request = $this->getRequest()->request;
         $requestParameters = $request->all();
         $requestParameters['flow_order_step'] = $step->getNumber();
-        $requestParameters[$step->getFormType()->getName()] = $stepData;
+        $requestParameters[$step->getFormType()] = $stepData;
```

Comment:
Hello @jDolba,

I have reviewed your PR and I found one problem. `$step->getFormType()` can return `FormTypeInterface`. You cannot use interface as key for an array. Can you find some better way to fix this?

Thank you.

Reply:
`FormTypeInterface` does not have any method to get "name" since symfony3 


also look at the
`\Shopsys\ShopBundle\Form\Front\Order\OrderFlow::loadStepsConfig`
which is used to set `$step::formType` - you will find that configuration of `form_type` as always string


also look at the issue and watch my link to symfony component commit where is also note in `@deprecation` note that you should always use class name (string)

``` Deprecated since Symfony 2.8, to be removed in Symfony 3.0.  Use the fully-qualified class name of the type instead. ```